imagery-cesium/plugins/entities-visualizer/base-entities-visualizer.ts

```diff
@@ -19,21 +18,39 @@ import {
 	FeatureCollection
 } from 'geojson';
 import {
-	Color,
 	CustomDataSource,
 	Entity,
 	BillboardGraphics,
 	PointGraphics,
 	PolylineGraphics,
 	PolygonGraphics,
-	ConstantProperty
+	ConstantProperty,
+	BoundingSphere,
+	PolygonHierarchy,
+	ColorMaterialProperty,
+	LabelGraphics,
+	HorizontalOrigin,
+	VerticalOrigin,
+	LabelStyle
 } from 'cesium'
 import * as geoToCesium from '../utils/geoToCesium';
 
 import { merge } from 'lodash';
-import { PolylineArrowMaterialProperty } from 'cesium';
 import { AnnotationMode } from '../../models/annotation-mode.enum';
-
+import {
+	getMarkerColor,
+	getMarkerSize,
+	getLineMaterial,
+	getStrokeWidth,
+	getShowOutline,
+	getStrokeColor,
+	getShowFill,
+	getFillColor,
+	getColor
+} from '../helpers/visualizer-style-helper';
+
+// TODO => remove using of declare const Cesium after having solved the issue
+// of new Cesium.CustomDataSource(dataSourceGuid) in getOrCreateDataSource mehod
 declare const Cesium: any;
```

Comment:
remove?

Reply:
when getting rid of the global Cesium variable, everything works fine except the "Point" annotation, and it happens only when refactoring the code from `new Cesium.CustomDataSource(dataSourceGuid)` to `new .CustomDataSource(dataSourceGuid)` in "getOrCreateDataSource" method